mlflow/server/js/src/shared/building_blocks/PageHeader.jsx

```diff
@@ -11,7 +11,13 @@ import { PreviewIcon } from './PreviewIcon';
 import { Spacer } from './Spacer';
 
 // Note: this button has a different size from normal AntD buttons.
-export { Button as HeaderButton };
+export function HeaderButton({ onClick, children, ...props }){
+  return (
+    <Button size='default' onClick={onClick} {...props} >
+      {children}
+    </Button>
+  )
+};
```

Comment:
I think ideally we wouldn't touch this component (and sizes below)since it's shared across many different pages

Also, this amendment doesn't seem to do anything extra really, since according to https://ant.design/components/button/, `default` is not a valid size.

Reply:
@sunishsheth2009, @xanderwebs, reverted the change to the export of HeaderButton (also removing the `size='default'` part).

Wrt to the changes in the CSS part: I thought this would be the best way to make the design adhere to the sketch provided by @tytownley. But now indeed this change is reflected on the other pages that make use of the `PageHeader` component too. Should we add some separate styling that only applies to the PageHeader in the ExperimentView? Or is it alright to have the same style changes on the other pages that make use of the PageHeader?

The main effect of the styling is that the headerbutton has a slightly different padding, and that the margin below the pagetitle is 16px instead of 8px.

Curious to hear your thoughts!

Reply:
@marijncv Could we do something that specifically affects the Share button? I think adding scoped global styles on `button`s would be potentially annoying to deal with if we have any other buttons in the header.

Maybe something like 
```jsx
function ShareButton(props) {
   return <HeaderButton className={css(...buttonOverrides)} {...props} />;
}
```

Reply:
@xanderwebs thanks for the suggestion! In the latest commit I've removed the style changes in the `PageHeader` component and created a local style definition for the sharebutton only in the `ExperimentView` component